src/main/java/com/ververica/platform/FlinkPullRequestsToKafka.java

```diff
@@ -0,0 +1,107 @@
+package com.ververica.platform;
+
+import static com.ververica.platform.io.source.GithubSource.EVALUATION_ZONE;
+import static java.time.format.DateTimeFormatter.ISO_LOCAL_DATE;
+import static java.time.format.DateTimeFormatter.ISO_LOCAL_TIME;
+
+import com.ververica.platform.entities.PullRequest;
+import com.ververica.platform.io.source.GithubPullRequestSource;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeFormatterBuilder;
+import java.time.temporal.ChronoField;
+import org.apache.flink.api.java.utils.ParameterTool;
+import org.apache.flink.streaming.api.datastream.DataStream;
+import org.apache.flink.streaming.api.environment.StreamExecutionEnvironment;
+import org.apache.flink.table.api.EnvironmentSettings;
+import org.apache.flink.table.api.bridge.java.StreamTableEnvironment;
+
+public class FlinkPullRequestsToKafka {
```

Comment:
Could you add a small javadoc to this class stating what it describing what the app does?